build/installer.nsh

```diff
@@ -41,6 +41,24 @@
     Abort "Unexpected error installing Windows subsystem for Linux: $R0"
   ${EndIf}
 
+  # Migrate WSL distro name from 'k3s' to 'rancher-desktop'
+  DetailPrint "Migrating from previous versions: renaming wsl distribution"
+  File "/oname=$PLUGINSDIR\migrate-wsl-from-v020.ps1" "${BUILD_RESOURCES_DIR}\migrate-wsl-from-v020.ps1"
+  nsExec::ExecToLog 'powershell.exe \
+    -NoProfile -NonInteractive -ExecutionPolicy RemoteSigned \
+    -File "$PLUGINSDIR\migrate-wsl-from-v020.ps1"'
+  Pop $R0
+  ${If} $R0 == "error"
+    MessageBox MB_OK "Could not carry out migration to wsl distribution rename."
```

Comment:
```suggestion
    MessageBox MB_OK | MB_ICONEXCLAMATION "Could not carry out migration to wsl distribution rename."
```
Since this is an error that the user should read.  See [documentation](https://nsis.sourceforge.io/Reference/MessageBox).

Reply:
done